test/storage/garbage_collector_test.cpp

```diff
@@ -1225,14 +1239,14 @@ TEST_F(GarbageCollectorTests, DeleteTest2) {
   }
 }
 
+/*
+ * Consider performing GC on the version chain sorted from newest to oldest:
+ * txn1 <- active txn
```

Comment:
Consistency with the previous notation
```suggestion
 T1 <- active txn
```